test/tests/functional/pbs_offline_vnodes.py

```diff
@@ -374,10 +355,151 @@ def test_fail_action_startup_hook(self):
                            id=mom.shortname,
                            max_attempts=3, interval=2, attrop=PTL_AND)
 
-        # Restore original node setup for future test cases.
-        self.server.cleanup_jobs(extend='force')
-        self.server.manager(MGR_CMD_DELETE, NODE, id="@default",
-                            expect=True)
-        for m in self.moms.values():
-            self.server.manager(MGR_CMD_CREATE, NODE,
-                                id=m.shortname)
+    def test_pbsnodes_o_single_mom(self):
+        """
+        Offline a mom using pbsnodes -o.
+        Since it is the only mom, all vnodes reported by her
+        should also be offline.
+        """
+        single_mom = ""
+        single_mom = self.moms.values()[0]
+        self.create_multi_vnodes(1)
+        self.server.expect(NODE, {ATTR_NODE_state: 'free'},
+                           id=single_mom.shortname, max_attempts=3,
+                           interval=2)
+
+        mom_host = single_mom.shortname
+        pbs_exec = self.server.pbs_conf['PBS_EXEC']
+        pbsnodes_cmd = os.path.join(pbs_exec, 'bin', 'pbsnodes')
+        pbsnodes_offline = pbsnodes_cmd + ' -o ' + mom_host
+        self.du.run_cmd(self.server.hostname, cmd=pbsnodes_offline, sudo=True)

```

Comment:
run_cmd() works better with a list of arguments, not a string. Pass pbsnodes_offline = [pbsnodes_cmd, '-o', mom_host]

Reply:
ok.  will update them.